operators/katib-controller/src/webhooks.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Do you need to have your own YAML for the webhooks or you can use it from the source: `/manifest/v1beta1/...` ?

Reply:
See https://github.com/canonical/charmcraft/issues/260 for why we can't symlink it to the existing file. If you'd like, I could take a look at adding a Makefile or similar that would copy the file over before calling `charmcraft build`.

Reply:
Got it, thank you for the information.
Maybe we can include this file copy in the script that I mentioned here: https://github.com/kubeflow/katib/pull/1465#discussion_r592518581.

Thus, this script will take care about changes between operator manifests and the Katib original manifests.
WDYT @knkski ?

We can add this script in the future PRs.

Reply:
Sure, sounds good :+1: